typings/index.d.ts

```diff
@@ -0,0 +1,1909 @@
+/**
+ * Type definitions for "material-ui" v1.0.0-beta.2
+ * Project: https://github.com/callemall/material-ui/
+ * TypeScript Version: 2.4
+ */
+
+///<reference types="react" />
+///<reference types="enzyme" />
+
+declare namespace MaterialUI {
```

Comment:
The reason why I used a namespace is b/c I didn't want to import the specialized component I created for JSS-styled components, like:

```ts
export default class Button extends MaterialUI.Component<ButtinProps> {}
```

Every module would have to import that declaration otherwise.